features/shadow-parts.yml

```
@@ -0,0 +1,11 @@
+name: Shadow Parts
+description: The `part` and `exportparts` HTML attributes expose elements of a shadow DOM as named parts. The `::part()` CSS pseudo-element selects named parts for styling.
```

```suggestion
description: The `part` and `exportparts` HTML attributes expose elements of a shadow DOM as named parts, which can be selected by the `::part()` CSS pseudo-element for styling.
```
I'm uncertain if this is an improvement- I'll leave it up to you.

---

I had a similar phrasing originally, so 👍